roles/nova-common/tasks/main.yml

```diff
@@ -29,3 +29,6 @@
   tags:
     - logrotate
     - logging
+
+- name: install virt-top
+  apt: pkg=virt-top
```

Comment:
Adding to the list of packages makes a lot of sense.  I'm not sure why I didn't do that in the first place.  PR update incoming